linera-service/Cargo.toml

```diff
@@ -103,7 +104,7 @@ fungible.workspace = true
 linera-base = { workspace = true, features = ["test"] }
 linera-chain = { workspace = true, features = ["test"] }
 linera-core = { workspace = true, features = ["test"] }
-linera-ethereum = { workspace = true }
+linera-ethereum.workspace = true
```

Comment:
Should this be an optional dependency, and only enabled with the `ethereum` feature?

Reply:
It cannot be as it is part of the `dev-dependency` where optional features are not allowed.